docs/reference/api-v1alpha.md

```diff
@@ -286,6 +286,33 @@ curl -i -X POST -H "Authorization: Token {api_token}" \
         "https://{org_name}.semaphoreci.com/api/v1alpha/plumber-workflows/:workflow_id/terminate"
 ```
 
+### Validating a workflow YAML
```

Comment:
Technically, this is validating a `pipeline` YAML since it only validates that one file and does not follow through the promotion chain. Can you move it to the `Pipelines`  section and also change `workflow` -> `pipeline` on line 297?

Reply:
Done.